python/marvin/web/templates/header.html

```diff
@@ -1,6 +1,35 @@
 
 {% from 'macros.html' import galidsearch, login %}
 
+
+<!-- Begin Cookie Consent plugin by Silktide - http://silktide.com/cookieconsent.
+We use it to show a banner if the used browser does not play well with Marvin.-->
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
+<script>
+window.addEventListener("load", function(){
+    if (navigator.userAgent.indexOf('Safari') != -1 && navigator.userAgent.indexOf('Chrome') == -1) {
+        window.cookieconsent.initialise({
```

Comment:
can we turn this cookieconsent intialization into a general banner creation, and pass in as variables, the message, dismiss, and href options content?